### ISSUE TITLE
An optoin to connect directly to Intiface Desktop

### ISSUE THREAD

**Issue by ba-rgn** [buttplugio/buttplug-unity#12]
A way to connect directly to an Intiface Desktop server without needing IntifaceCLI would be a nice addition.
One reason for this is that it would allow Linux/WINE users to play games that use this plugin, as currently IntifaceCLI crashes when run under WINE.

**Comment by WeaselKing** [comment]
Ha! I was just about to leave a similar issue for an identical reason! Making IntifaceCLI work under WINE or Proton would also solve the issue of course <3 Not sure what's going on there. Proton doesn't record the crash happen which isn't particularly useful for development, but there might be a way to fish out why it's failing.

**Comment by qdot** [comment]
Yeah, honestly I'm... not actually happy with the FFI thing myself. It was an interesting experiment but has ultimately ended up being more trouble than its worth.

There's actually sort of a solution to this already:

https://github.com/Er1807/ManagedButtplugIo

Someone wrote a completely Managed/C# version of the FFI API. I'm probably just going to swap something like this in and always have Unity call out to Intiface Desktop/Central (it's currently getting renamed from Desktop to Central heh), because trying to maintain the executable version, deal with Mono and the FFI, etc has just been way too difficult.
